inference/enterprise/helm-chart/roboflow-inference-server/values-cpu.yaml

```diff
@@ -0,0 +1,85 @@
+# Default values for roboflow-inference-server.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+# Type of environment - can be cpu or gpu
+computeType: cpu
+
+image:
+  repository: roboflow/roboflow-inference-server-cpu
+  tag: 0.9.7
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Environment variables to set
+# More information: 
+# https://inference.roboflow.com/quickstart/docker_configuration_options/
+envMap:
+  - name: NUM_WORKERS
+    value: 1
+  - name: DYNAMIC_ENVIRONMENT
+    value: "true"
+  - name: LOG_LEVEL
+    value: "DEBUG"
+  - name: METRICS_ENABLED
+    value: "false"
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+# minAvailablePods: '80%' ## Commented because we removed pod-disruption-budget
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 9001
+
+ingress:
+  enabled: false
+
+resources: 
+  limits:
+    cpu: 2000m
+    memory: 4Gi
+  # Add the nvidia gpu resource if you run this on nodes containing nvidia GPUs with the 
+  # corresponding roboflow/roboflow-inference-server-gpu image (replace above)  
+  # nvidia.com/gpu: 1
+  requests:
+    cpu: 1000m
+    memory: 1Gi
```

Comment:
since memory is not compressible - 1G may not be enough (we allow multi-model load per worker - reasonable would be to have higher request
just checked that loading a small model takes ~500M of RAM

Reply:
The request is the lower bound, the upper bound is limit. That is set higher in this file.